.travis.yml

```diff
@@ -1,31 +0,0 @@
-os: linux
-language: node_js
-node_js:
-- '14'
-addons:
-  chrome: stable
-before_install: "source scripts/helpers.sh"
-install: yarn install
-before_script: "yarn build"
-script:
-  - "./scripts/travis.sh"
-dist: xenial
-env:
-  global:
-  - NOKOGIRI_USE_SYSTEM_LIBRARIES=true
-  - SAUCE_BROWSER_NAME="chrome"
-  - SAUCE_BROWSER_VERSION=""
-  - SAUCE_PLATFORM="Windows 10"
-  - SAUCE_APPIUM_VERSION=""
-  - SAUCE_DEVICE_NAME=""
-  - SAUCE_DEVICE_ORIENTATION=""
-  - SAUCE_PLATFORM_VERSION=""
-  - SAUCE_PLATFORM_NAME=""
-  - secure: RW99TW7YvGgWt/P8fQQziPbxNITaJMO/4q2iCX72u/PtTu2dkeaCqUiRUi7qyV8jIIk7U4oruxJRFQpTaCIX5+JWZsSwLskoTsspOI9aTzyPdhVNckM1vpA5VfwstdyQTkX7zzKPLdvXK0xj0ZzQ+ACH/SobQNYKiAMvW7FVkIc=
-  - secure: gFSORYQLSBNIK8peGgUOhd0Yj6GzE18HA0GLrmp0PMtobe003poalkNgLsPy2W2QFZbG3bCjbqFMmCbUh5uRnX8eCy/whe+lwXeQsv5uh0EIGWgxVxhk79dCx5cakERusoj2BUgJJ432R7nRXjEW5v2AK9QAOeCG2Dvwdb8WSes=
```

Comment:
Could not find usage of these anywhere. Asked around and no one knows if this was being used or not.